boards/microbit_v2/Makefile

```
@@ -15,12 +17,34 @@ TOCKLOADER=tockloader
 .PHONY: install
 install: flash
 
+
 .PHONY: flash-debug
 flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
-	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $<; verify_image $<;  reset; shutdown;"
+		$(CARGO) flash --chip nRF52833_xxAA --verify --path $<
```

```suggestion
	$(CARGO) flash --chip nRF52833_xxAA --verify --path $<
```

This indentation seems off, no?